### DISCUSSION TITLE
The preview for ReST README is not loading

### DISCUSSION THREAD

**Discussion by pandafy** [community/community#89118] [Repositories]
### Select Topic Area

Bug

### Body

This is occurring on multiple repositories of @openwisp (e.g. https://github.com/openwisp/openwisp-monitoring). The preview for README.rst is not loading. 

![Screenshot from 2024-01-19 14-12-10](https://github.com/community/community/assets/32094433/ba658499-8851-4ae0-8cec-8f1a1ee6372c)

**Comment by ghost** [comment]
This is because the README.rst file is enormously big. Since rst files are super powerful and as many functionalities that need processing... It takes too long to load and github doesn't display. You can verify this by seeing the RAW file. So this can be solved if you have a very fast network, even i have a 5g connection but i cannot see it...Or compress the size of the file but really hard to do as their READMEs are very large....If this was .md file it wont have this much functionalities but it would take less time to load...
